bonsai/en.lproj/Localizable.strings

```diff
@@ -6,6 +6,15 @@
   
 */
 
+"Onboarding_1_title" = "Welcome!";
+"Onboarding_1_description" = "Bonsai represents the money tree. Set financial goals by creating a budget, make transactions, keep track of expenses and income.";
```

Comment:
I would suggest to edit middle part of the text:
"Set financial goals, track your money flow, notice your financial habits"